src/bots/discord.ts

```diff
@@ -55,8 +55,11 @@ export default class DiscordBot extends BotClient {
   }
 
   public async getUser(): Promise<User> {
-    const userID = this.bot.user.id;
-    return new User(this, userID);
+    const userID = this.bot.user?.id;
+
+    if (!userID) this.logger.error('Bot user not found.');
+
+    return new User(this, userID ?? '-1');
```

Comment:
Would be nice if -1 was documented as a not-found value in JSDoc for this function

Reply:
I changed it to throw an error instead to avoid weird bugs